diff --git a/arduino.c b/arduino.c
index 123e83e..75418c9 100644
--- a/arduino.c
+++ b/arduino.c
@@ -92,10 +92,10 @@ static int arduino_open(PROGRAMMER * pgm, char * port)
 
   /* Clear DTR and RTS to unload the RESET capacitor 
    * (for example in Arduino) */
-  serial_set_dtr_rts(&pgm->fd, 0);
+  serial_set_dtr_rts(&pgm->fd, 1);
   usleep(50*1000);
   /* Set DTR and RTS back to high */
-  serial_set_dtr_rts(&pgm->fd, 1);
+  serial_set_dtr_rts(&pgm->fd, 0);
   usleep(50*1000);
 
   /*
